### ISSUE TITLE
Train Blips

### ISSUE THREAD

**Issue by Stormshadow333** [Thunderstorm441/ts-trains-redm#1]
Hey is this possible to add blips for trains on map? Thankyou for your help

**Comment by Nathant995** [comment]
> Hey is this possible to add blips for trains on map? Thankyou for your help

Yes, just find the hash for the train blip(look on the released hashes db) and attach it to the train object :)
